modules/arm_plugin/src/arm_converter/arm_converter_mat_mul.cpp

```diff
@@ -3,21 +3,207 @@
 //
 
 
+#include <src/cpu/kernels/CpuConvertQuantizedSignednessKernel.h>
+#include <arm_compute/runtime/NEON/NEScheduler.h>
 #include <arm_compute/runtime/NEON/functions/NEFullyConnectedLayer.h>
 #include "arm_converter/arm_converter.hpp"
 
 namespace ArmPlugin {
 enum InputArg {Features, Weights, Bias};
+struct NEFullyConnectedLayerQI final: public arm_compute::IFunction {
```

Comment:
Can we move this implementation to a separate subfolder? 
Can we generalize some code inside them? Makes some base `LayerQI` class with some common parts of code?
It will be good if these parts were moved into separate functions with clear names.

Reply:
I think these wrappers shouldn't be separated from the conversions they are used in. At least such a structure was used for all other wrappers introduced to support reference based operations. But I definitely agree with simplification of wrappers via generalized class.
However I suppose that refactoring could take some time so I prefer to do it as a part of separate PR to improve LPT support ASAP and avoid possible merge conflicts due to the massiveness of this PR code change.